Subscriber/CheckoutSubscriber.php

```diff
@@ -15,6 +16,7 @@
 use AdyenPayment\Components\Manager\AdyenManager;
 use AdyenPayment\Components\PaymentMethodService as ShopwarePaymentMethodService;
 use AdyenPayment\AdyenPayment;
+use Shopware\Bundle\StoreFrontBundle\Struct\Attribute;
```

Comment:
These new `use` statements are not used (pun intended), right?

Reply:
haha nice one,  
it's already covered in an internal review so with an update of the PR this is corrected.